random.c

```diff
@@ -0,0 +1,29 @@
+/*Matthew Danielson
+* 6/29/16
+* Random.c
+*Generates a random number to be generated
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main(){
+	srand(time(NULL));
+	int randint = rand();
+	int randint2 = randint%10 ;
```

Comment:
No need to create a second variable - you could/should have reused randint. Not doing so wastes memory.